calculators/ItemNetworthCalculator.js

```diff
@@ -23,29 +23,34 @@ const { reforges } = require('../constants/reforges');
 const { getHypixelItemInformationFromId } = require('../constants/itemsMap');
 
 const networthManager = require('./NetworthManager');
+const { Helper } = require('./helper/functions');
+const { HotPotatoBookCalculation } = require('./items/HotPotatoBook');
+const { Recombobulator3000Calculation } = require('./items/Recombobulator_3000');
+const { PickonimbusCalculation } = require('./items/Pickonimbus');
 
-class ItemNetworthCalculator {
+class ItemNetworthCalculator extends Helper {
```

Comment:
Why extend `Helper`? What else is `Helper` used for?

Reply:
Mostly to clean out the main fie, it has getItemid(), getItemName() and basic checks